crates/nu-engine/src/evaluation_context.rs

```diff
@@ -188,30 +188,8 @@ impl EvaluationContext {
     /// If an error occurs while reloading the config:
     ///     The config is not reloaded
     ///     The error is returned
-    pub fn reload_config(&self, cfg_path: &ConfigPath) -> Result<(), ShellError> {
```

Comment:
We now pass NuConfig instead of Path, so the next 20 lines error handling is obsolete